aten/src/ATen/NestedTensorImpl.cpp

```diff
@@ -276,6 +276,9 @@ c10::SymInt NestedTensorImpl::sym_numel_custom() const {
 bool NestedTensorImpl::is_contiguous_custom(MemoryFormat) const {
   return nested_tensor_impl_is_contiguous(this);
 }
+bool NestedTensorImpl::definitely_contiguous_fast_custom(MemoryFormat format) const {
```

Comment:
in those situations i am just calling is_contiguous_custom we can always come back and have special implementations if we hit dde in those.